blog/2020-10-23-the-making-of-new-user-sdk-tutorials.md

```diff
@@ -0,0 +1,94 @@
+
+- [Go money transfer template repo](https://github.com/temporalio/money-transfer-project-template-go)
+- [Go hello world template repo](https://github.com/temporalio/hello-world-project-template-go)
+- [Java money transfer template repo](https://github.com/temporalio/money-transfer-project-template-java)
+- [Java hello world template repo](https://github.com/temporalio/hello-world-project-template-java)
+
+## The tutorial
+
+<img class="docs-image-centered docs-image-max-width-20" src={require('../static/img/docs/tutorials.png').default} />
+
+&nbsp;&nbsp; Since we really wanted new users to grasp some of the core value propositions Temporal offers, we decided to aim for a “tutorial” style of documentation. The first iterations aimed to build upon and replace the existing SDK “quick start” pages that maintained the industry standard of showing an example that printed “Hello World!” to the console. But, as you may have guessed, this route wasn’t sufficient enough to show users the real values that Temporal offers.
+
+So, we approached our co-founder and CEO, Maxim Fateev, and asked him to give us the same demonstration that he liked to give when introducing the software to another engineer for the first time. One of Maxim’s go-to scenarios is a money transfer from one bank account to another. “Aha!” we said, because as it turns out, for someone new to Temporal the attraction doesn’t come from using the SDK to print “Hello World!”, but it is instead to witness the inherent benefits that Temporal offers by running simulations using a pre-built application.
+
+Because everyone can understand the ramifications of losing money during a transaction, the money transfer demonstration shows off at least four of Temporal’s main value propositions:
+
+1. The state of the code execution is maintained even through server outages and crashes.
+2. There is deep visibility into the state of the code execution via the CLI or UI.
+3. Code has automatic timeouts and retries, which are easily configurable.
+4. Bugs can be fixed “on-the-fly” without losing the state of the code execution.
+
+So, it was this direction that we decided to send new users. Once a user can wrap their heads around the value that Temporal brings to their application right out of the box, then spending time and energy on application setup and learning the SDK becomes trivial.
+
+## Next steps
+
+<img class="docs-image-centered docs-image-max-width-20" src={require('../static/img/docs/steps.png').default} />
+
+&nbsp;&nbsp; We are far from satisfied with our documentation. We plan to bring SDK reference right into the docs, merge SDK tutorials into a single journey, so that a user can easily see how things are done across different languages, rewrite our concepts, re-write server management docs, and prepare for our hosted cloud offering that will enable all developers to build invincible applications.
```

Comment:
```suggestion
&nbsp;&nbsp; We are far from satisfied with our documentation. We plan to bring SDK reference right into the docs, merge SDK tutorials into a single journey, so that a user can easily see how things are done across different languages, rewrite our concepts, re-write server management docs, and prepare for our hosted cloud offering that will enable all developers to build invincible applications.
```

I think you can say we still need to do lots of improvements without saying "We are far from satisfied". I also wouldn't publicly disclose our specific plans "We plan to bring SDK reference right into the docs, merge SDK tutorials into a single journey," just in case you realize its not a good idea.